src/tests/JIT/Regression/JitBlue/Runtime_62692/Runtime_62692_r.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <Optimize>False</Optimize>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <DisableProjectBuild>true</DisableProjectBuild>
```

Comment:
This is disabling the test completely. We do not want that - we want to keep the test enabled outside Mono.

The right place to disable the tests for Mono is around 
 https://github.com/dotnet/runtime/blob/main/src/tests/issues.targets#L1171-L1174